tests/torch/test_pytorch_patch.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from nncf.torch.dynamic_graph.patch_pytorch import PrivateFunctionsToPatch
+from nncf.torch.graph.operator_metatypes import PT_OPERATOR_METATYPES
+
+
+def test_is_any_metatype_op_names_duplicated():
+    op_name_list = set()
+    for operator in PT_OPERATOR_METATYPES.registry_dict:
+        for op_name in PT_OPERATOR_METATYPES.get(operator).op_names:
+            assert op_name not in op_name_list
+            op_name_list.add(op_name)
```

Comment:
Updated test according to the comments.
But this test actually is never tested, because when we try to add a new metatype with the duplicated operator name to Registry, it will lead to an runtimerror.

Reply:
Done